.eslintrc.json

```diff
@@ -72,11 +73,6 @@
     // 7.10 https://github.com/airbnb/javascript#functions--constructor
     "no-new-func": "error",
 
-    // 7.11 https://github.com/airbnb/javascript#functions--signature-spacing
-    // 19.2 https://github.com/airbnb/javascript#whitespace--before-blocks
-    "space-before-function-paren": ["error", "never"],
-    "space-before-blocks": ["error", "always"],
```

Comment:
As discussed on another PR, let's add those back in and configure them not to complain about async arrow functions.

Reply:
Restored and reconfigured rules